### ISSUE TITLE
Project 100% finished but will not be delivered

### ISSUE THREAD

**Issue by Voosle** [StrangeLoopGames/EcoIssues#5929]
After crafting projects up to 100% the items will not be delivered to the linked chest and no 'Collect' button apears.
Occured at workbench and campfire.

![bug - finished project item not delivered](https://user-images.githubusercontent.com/22961978/37259122-c3f76962-2582-11e8-80ba-a2b4223bc417.jpg)

**Comment by spool1** [comment]
This is a devastating issue that needs to be looked into by the devs. It is not just happening to the workbench/campfire, but any workstation. It has been leading to thousands of resources being deleted essentially as the orders get stuck at 100% 100%, and done. 

Our server has been left in a state scared of crafting Mortared Stone in anything more than 10 blocks at a time because well over 1000+ mortared stone has been lost due to this bug. Even the small 10x orders get blocked by this bug and cause the resources to be deleted since the ONLY option to free up the workstation is to "Cancel Project" 

It may be related to a workstation that has a lot of chests and stockpiles in its linked ranged, whether or not the stockpile is ticked to be linked. It seems almost random at this stage but we are trying to get repo steps.

**Comment by Voosle** [comment]
Hi,
thanks for this quick feedback! :-)

Waiting for the bugfix.

**Comment by MittenMarauder** [comment]
I have also experienced this issue. Unable to consistently reproduce on our server, but does occur sporadically.

**Comment by MittenMarauder** [comment]
Our server is also experiencing an issue where the stockpiles are eating materials on occasion. It makes me wonder if there is something related going on.

**Comment by spool1** [comment]
Yeah, I believe that is related to moving things one from the stockpile to another, if you move an entire full stockpile from one to another the other stockpile sometimes won't be full I did see a bug report for that on this Github page as well. One assumption is that it only tries to output stock into something once and if it chooses to try put it into a full stockpile/chest, or a stockpile/chest that could not take the full amount of items, that it fails and doesn't try to find another and bugs out. But we're had situations where even that seems to not be true.

All we need to always have the ability to remove things from a "Done" job into our hands/backpack. At least it would give us a way to solve the issue.

**Comment by MarjieAVolk** [comment]
Closing as duplicate of https://github.com/StrangeLoopGames/EcoIssues/issues/3240, thanks for all the details

**Comment by MarjieAVolk** [comment]
Actually, if anybody has a save file where this is currently happening, could you send it to me?

**Comment by Karmek** [comment]
Hi, It happened to us as well today. Food got stuck in Campfire. The collect button did not show up. I asked a friend to check if he could remove the items, same there.

However a third friend had the collect button! And he was able to claim the stuck items from my campfire. So perhaps it is some kind of odd permission problem?
